src/c3_fmt.c

```diff
@@ -211,7 +211,7 @@ static void init(struct fmt_main *self)
 			salt = options.subformat;
 			/* turn off many salts test, since we are not updating the */
 			/* params.tests structure data.                            */
-			self->params.benchmark_length = -1;
+			self->params.benchmark_length = 0x107;
```

Comment:
Maybe `self->params.benchmark_length |= 0x100;` would be friendlier to further maintenance.

Reply:
Agreed (although I'm just trying to merge now :)